docs/ios/v2/how-to-guides/extend-capabilities/plugins/noise-cancellation.mdx

```diff
@@ -29,6 +29,11 @@ let pathForNCModel = HMSNoiseCancellationModels.path(for: .smallFullBand)
 ## Minimum Requirements
 -   Minimum 100ms SDK version required is `1.7.0`
 
+<Note type='warning'>
+**IMPORTANT** <br/>
+Enable Noise Cancellation in the template configuration. Learn more on enabling this feature from [here](/get-started/v2/get-started/features/noise-cancellation#enabling-the-noise-cancellation)
```

Comment:
Sentence update:

Learn more about enabling this feature [here](/get-started/v2/get-started/features/noise-cancellation#enabling-the-noise-cancellation).